src/test/java/io/confluent/idesidecar/restapi/util/CPDemoTestEnvironment.java

```diff
@@ -0,0 +1,113 @@
+package io.confluent.idesidecar.restapi.util;
+
+import io.confluent.idesidecar.restapi.models.ConnectionSpec;
+import io.confluent.idesidecar.restapi.util.cpdemo.CPServerContainer;
+import io.confluent.idesidecar.restapi.util.cpdemo.OpenldapContainer;
+import io.confluent.idesidecar.restapi.util.cpdemo.SchemaRegistryContainer;
+import io.confluent.idesidecar.restapi.util.cpdemo.ZookeeperContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.util.Optional;
+
+/**
+ * A {@link TestEnvironment} that starts a CP Demo environment with a two-node Kafka cluster,
+ * Zookeeper, OpenLDAP, and Schema Registry.
```

Comment:
For the sake of faster tests and a better memory footprint, what about running a **one**-node Kafka cluster in [KRaft mode](https://docs.confluent.io/platform/current/installation/docker/config-reference.html#required-ak-configurations-for-kraft-mode)?

https://docs.confluent.io/platform/current/installation/docker/config-reference.html#required-ak-configurations-for-zk-mode states that `As of Confluent Platform 7.5, ZooKeeper is deprecated for new deployments.`

Reply:
We can do this in a followup.